byron-proxy/src/exec/Logging.hs

```diff
@@ -52,10 +53,18 @@ convertTrace trace = case trace of
                        , Monitoring.severity = sev
                        , Monitoring.privacy = Monitoring.Public
                        }
-        logContent = Monitoring.LogMessage text
+        logContent = content -- Monitoring.LogMessage text
         logObject  = Monitoring.LogObject name logMeta logContent
     f logObject
```

Comment:
@CodiePP is there something in iohk-monitoring that does this for me: gathers the thread id and current time the puts it in the appropriate places? Or is the user of iohk-monitoring expected to do this, as I've done here?

Reply:
`mkLOMeta`